### PR TITLE
Minikube-tutorial

### PR DESCRIPTION
**Pull request by bcsherma**
## Description

What does the pull request do? If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Ticket

Does this PR fix an existing issue? If yes, provide a link to the ticket here:

## Checklist

Check if your PR fulfills the following requirements. Put an `X` in the boxes that apply.

- [ ] Files I edited were previewed on my local development server with `yarn start`. My changes did not break the local preview.
- [ ] Build (`yarn docusaurus build`) was run locally and successfully without errors or warnings.
- [ ] I merged the latest changes from `main` into my feature branch before submitting this PR.

### CODE REVIEW

**Comment by cvphelps** [comment]
@ngrayluna Ben is making some edits here, so this one can wait until next week when you're back in the office

File: docs/tutorials/minikube_gpu.md [hunk 1]
```diff
@@ -0,0 +1,400 @@
...
+
+If you want to setup a Minikube cluster on a cloud machine, we recommend you create a Kubernetes cluster with GPU support using your cloud provider’s tools. AWS, GCP, Azure, Coreweave, and other cloud providers have tools to create Kubernetes clusters with GPU support.
+
+If you want to set up a Minikube cluster for GPU scheduling on a machine with a single GPU, we would recommend you use a Launch [Docker queue](/guides/launch/setup-launch-docker) instead. You can still follow the tutorial for fun, but the GPU scheduling will not be very useful.
+
+:::
+
+## Background
+
+If you use Docker queues for workloads running on a single host, there is a compelling reason to use a Kubernetes queue with an agent connected to a local Kubernetes cluster instead: GPU scheduling. Docker queues control how the job is run through `docker run` arguments, which only allow requesting specific GPU on the host or `all` which severely limits the efficiency of parallel workloads. Kubernetes allows you to specify a number of desired GPU and then takes care of the scheduling.
```
Comment (ngrayluna) [inline docs/tutorials/minikube_gpu.md]: This intro paragraph was a bit confusing. Can you re word it, simplify, condense it?

**Review by ngrayluna — CHANGES_REQUESTED** [review]
@bcsherma Did an edit pass and pushed said edits to your branch. Can you review and edit the comment I made? When we get that in, we can merge and ship it.

**Comment by tristan-spaulding2** [comment]
If possible and there are no blocking comments, could we ship this update today / tomorrow?  This guide is quite powerful and I'm working with marketing to do a small push that would direct attention to our support here.  Thanks.

**Comment by bcsherma** [comment]
> If possible and there are no blocking comments, could we ship this update today / tomorrow? This guide is quite powerful and I'm working with marketing to do a small push that would direct attention to our support here. Thanks.

Yep, I think I just need to address this one comment from @ngrayluna

File: docs/tutorials/minikube_gpu.md (hunk 1 above)
Comment (bcsherma) [reply docs/tutorials/minikube_gpu.md]: Agreed, not my finest work xD 

I've reworked it, lmk what you think

**Review by ngrayluna — APPROVED** [review]

**Pull request merged**